apostrophecms/apostrophe
# Pro 2730 groups in relationships

[PRO-2730](https://linear.app/apostrophecms/issue/PRO-2730/manage-groups-in-relationship-fields)

## Summary

Allows fields grouping in relationships schemas. share existing code between `AposDocEditor` and `AposRelationshipEditor` through mixins (😢).
Errors are handled the same way.

## What are the specific steps to test this change?

* Run an A3 project and link this branch
* Check that grouping is still working the same when editing docs
* Adds schema fields to a relationship and group them.
* Check that when editing the relationship the tabs are here and working
* In case of error (empty required field), check that we can't save the relationship and that we have a tooltip with remaining errors.
* In case of error, check that the number of errors is marked on each concerned tab.

## What kind of change does this PR introduce?

## Make sure the PR fulfills these requirements:
